docs/admin/install/steps/install-python.rst

```diff
@@ -6,27 +6,32 @@ Python modules
    We're using virtualenv to install Weblate in a separate environment from your
    system. If you are not familiar with it, check virtualenv :doc:`venv:user_guide`.
 
+.. warning::
+
+   Run the following commands in an empty directory
 
 #. Create the virtualenv for Weblate:
 
    .. code-block:: sh
 
-        virtualenv --python=python3 ~/weblate-env
+        python3 -m venv .
 
 #. Activate the virtualenv for Weblate:
 
    .. code-block:: sh
 
-        . ~/weblate-env/bin/activate
+        source bin/activate
 
 #. Install Weblate including all optional dependencies:
 
    .. code-block:: sh
 
         # pkgconfig is needed to install borgbackup 1.2
-        pip install pkgconfig
+        pip3 install pkgconfig
+        # mysqlclient is needed if you want use mysql/mariadb
+        pip3 install mysqlclient
```

Comment:
The recommended setup is using PostgreSQL. Still, if you want to use Weblate with MySQL, it is better to install it using `pip install Weblate[all,MySQL]`, that way you will get whatever is needed to run Weblate with MySQL.